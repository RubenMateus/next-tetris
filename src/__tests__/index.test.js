import { render, screen } from "@testing-library/react";
import App from "@/pages/index";

describe("App", () => {
  it("renders Start Game Button", () => {
    render(<App />);

    const startGameButton = screen.getByText(/start game/i);

    expect(startGameButton).toBeInTheDocument();
  });

  it("render snapshot", () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});