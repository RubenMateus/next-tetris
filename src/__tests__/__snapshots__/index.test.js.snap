// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="sc-bdvvtL hdrxtZ"
      role="button"
      tabindex="0"
    >
      <div
        class="sc-gsDKAQ hjIYNH"
      >
        <div
          class="sc-dkPtRN bzcvOx"
          height="20"
          width="12"
        >
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
          <div
            class="sc-hKwDye ePoxSY"
            color="0, 0, 0"
            type="0"
          />
        </div>
        <aside>
          <div>
            <div
              class="sc-eCImPb itejFv"
            >
              Score: 0
            </div>
            <div
              class="sc-eCImPb itejFv"
            >
              rows: 0
            </div>
            <div
              class="sc-eCImPb itejFv"
            >
              Level: 0
            </div>
          </div>
          <button
            class="sc-jRQBWg hXeCXg"
          >
            Start Game
          </button>
        </aside>
      </div>
    </div>
  </div>
</div>
`;
